src/lib.rs

```diff
@@ -448,16 +455,20 @@ struct Client {
 }
 
 impl Client {
-    async fn new(server_url: &str, http: Box<dyn HttpClient>) -> Result<Self, Error> {
+    async fn new(directory_url: &str, http: Box<dyn HttpClient>) -> Result<Self, Error> {
```

Comment:
This unrelated rename should be separately motivated and in a separate commit/PR.

Reply:
But I'll double check and log an issue for it.

Reply:
https://github.com/instant-labs/instant-acme/issues/53